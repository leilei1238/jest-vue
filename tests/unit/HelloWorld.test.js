import { shallowMount } from '@vue/test-utils'
import HelloWorld from '@/components/HelloWorld.vue'

describe('HelloWorld.vue', () => {
  // it('renders props.msg when passed', () => {
  //   const msg = 'new message'
  //   // shallowMount 浅渲染
  //   const wrapper = shallowMount(HelloWorld, {
  //     propsData: { msg }
  //   })
  //   expect(wrapper.vm.msg).toEqual('new message')
  //   console.log(wrapper.text())
  // })

  it('组件渲染正常', () => {
    const wrapper = shallowMount(HelloWorld, {
      propsData: { msg: 'helloworld' }
    })
    expect(wrapper).toMatchSnapshot()
  })
})
