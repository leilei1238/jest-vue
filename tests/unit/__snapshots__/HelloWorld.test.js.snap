// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.vue 组件渲染正常 1`] = `
<div class="hello">
  <h1>helloworld</h1>
  <p>22222</p>
</div>
`;
